services/ingestor/src/main/kotlin/company/thewebhook/ingestor/Application.kt

```diff
@@ -47,13 +50,19 @@ fun Application.module() = launch {
             .lowercase()
             .split(",")
 
+    val serverHostRegexString =
+        environment.config
+            .propertyOrNull("ingestor.regex.serverhost")?.getString()
+            ?: throw ServerHostRegexException("Serverhost regex string not provided")
+
     val producer by inject<Producer<ByteArray>>()
     producer.connect(messageStoreConfig)
 
     routing {
         HttpMethod.DefaultMethods.forEach {
             route("{...}", it) {
                 handle {
+                    val serverHostRegex = Regex(serverHostRegexString)
                     val serverHost = call.request.origin.serverHost
```

Comment:
@ankit-rawani also, server hostnames are case insensitive, but Kafka topic names are not. Can you convert this serverHost to lower case as well so that it is consistent?

Reply:
Done